resources/js/Root.jsx

```diff
@@ -0,0 +1,22 @@
+import React from "react";
+import { Box } from "@mui/material";
+import Navbar from "../components/Navbar";
+import { Outlet } from "react-router-dom";
+
+export default function Root() {
+    return (
+        <>
+            <Navbar />
+            <Box
+                sx={{
+                    paddingLeft: "1rem",
+                    paddingRight: "1rem",
+                    marginTop: "3rem",
+                }}
```

Comment:
essaie d'utiliser des nombre (voir mon commentaire précédent sur le spacing)

Reply:
replaced with this code `<Box paddingLeft={2} paddingRight={2} marginTop={6} id="detail">`